app/client/src/components/layers/changeStyle/IconVectorStyle.vue

```diff
@@ -3,17 +3,34 @@
     <v-tabs grow v-model="tab" style="width: 400px; margin:auto;">
       <v-tab :key="1">
         <v-badge>
-          <b>Change Icon Size</b>
+          <b>Color</b>
```

Comment:
Can we add this and other texts to the translation?

Reply:
Yes of course, I will add it now